src/UQpy/surrogates/polynomial_chaos/PolynomialChaosExpansion.py

```diff
@@ -52,6 +52,11 @@ def polynomials_number(self):
     def inputs_number(self):
         return self.polynomial_basis.inputs_number
 
+    def set_ed(self, x: np.ndarray, y: np.ndarray):
```

Comment:
Write meaningful function names. If the function needs to be public please write doctrings and add typing.